test/jasmine.utils.js

```diff
@@ -75,7 +75,13 @@ function acquireChart(config, options) {
 	wrapper.appendChild(canvas);
 	window.document.body.appendChild(wrapper);
 
-	chart = new Chart(canvas.getContext('2d'), config);
+	try {
+		chart = new Chart(canvas.getContext('2d'), config);
+	} catch (e) {
+		window.document.body.removeChild(wrapper);
```

Comment:
This [test](/chartjs/Chart.js/blob/2922dc96cfd3c3448538c19c02b349fe939f16eb/test/specs/core.controller.tests.js#L135) was leaking because the chart construction throws and the wrapper/canvas wasn't removed.